package.json

```diff
@@ -110,6 +110,7 @@
     "test": "cd frontend && mocha --require babel-register --require test-setup.js --recursive",
     "test:ci": "NODE_ENV=test nyc mocha --recursive --reporter mocha-junit-reporter --require frontend/test-setup.js frontend/test",
     "test:watch": "npm test -- --watch",
+    "test:all": "npm run lint && npm run test && cd addon && npm run lint && npm run coverage && npm run test", 
```

Comment:
i think we can just do `npm run coverage` for the addon. it runs all the other stuff too